src/tag.cpp

```diff
@@ -142,8 +144,9 @@ void HSTag::applyFloatingState(Client* client)
     needsRelayout_.emit();
 }
 
-void HSTag::setVisible(bool visible)
+void HSTag::setVisible(bool newVisible)
 {
+    visible = newVisible;
```

Comment:
Do we also need to check if visible already has the value of newVisible here?

Reply:
I'd say 'no' because we didn't check it before either and all the functions we call here have their own visibility flag anyway. Also, if we don't do any shortcuts here, we know that after `HSTag::setVisible()` the visibility-flags of all windows on this tag are in sync.

Reply:
Ok, makes sense, thanks!